app/components/RelatedRecordTable.tsx

```diff
@@ -21,11 +26,19 @@ export default function RelatedRecordTable<T extends Record<string, unknown>>({
   mutation,
   isValidating,
   title,
+  headerActionButton,
   onSaveCallback,
+  mutationVariables,
 }: RelatedRecordTableProps<T>) {
   return (
     <Card>
-      <Card.Header>{title}</Card.Header>
+      <Card.Header>
+        <div className="d-flex justify-content-between">
+
+        {title}
+        {headerActionButton && headerActionButton}
```

Comment:
this is checking if it exists and if so, render it? Trying to picture what would render if it was just `{headerActionButton}` or maybe it should be `{!!headerActionButton && headerActionButton}`?